src/containers/AssetList.js

```diff
@@ -1,62 +1,116 @@
 import {
   Table,
   TableBody,
   TableHeader,
   TableHeaderColumn,
   TableRow,
-  TableRowColumn,
-} from 'material-ui';
+} from 'material-ui/Table';
+import AssetListItem from '../components/AssetListItem';
 import AssetListHeader from '../components/AssetListHeader'
+import '../style/App.css';
+
+// Mock data until we can fetch data from the api
+const assetData = [
+	{
+		name: 'Asset #1',
+		status: 'In-progress',
+		department: 'UIS',
+		private: true,
+		lastedited: 'today'
+	},
+	{
+		name: 'Asset #2',
+		status: 'In-progress',
+		department: 'UIS',
+		private: true,
+		lastedited: 'today'
+	},
+	{
+		name: 'Asset #3',
+		status: 'Complete',
+		department: 'UIS',
+		private: true,
+		lastedited: 'today'
+	},
+	{
+		name: 'Asset #4',
+		status: 'In-progress',
+		department: 'UIS',
+		private: false,
+		lastedited: 'today'
+	},
+	{
+		name: 'Asset #5',
+		status: 'Complete',
+		department: 'UIS',
+		private: true,
+		lastedited: 'today'
+	},
+	{
+		name: 'Asset #6',
+		status: 'In-progress',
+		department: 'UIS',
+		private: true,
+		lastedited: 'today'
+	},
+	{
+		name: 'Asset #7',
+		status: 'In-progress',
+		department: 'UIS',
+		private: false,
+		lastedited: 'today'
+	},
+];
 
 const TITLES = {
   '/assets/dept': 'Assets: My department',
-  '/assets/edited': 'Assets: Edited recently',
   '/assets/all': 'Assets: All',
 };
 
-/*
-  Renders the list view of Assets - different filters can be applied to this view.
-  */
+
 const AssetList = ({ match }) => (
-  <div>
-    <AssetListHeader title={TITLES[match.url]} />
-    <div className="App-main">
-      <Table selectable={false}>
-        <TableHeader displaySelectAll={false}>
-          <TableRow>
-            <TableHeaderColumn>Name</TableHeaderColumn>
-            <TableHeaderColumn>Status</TableHeaderColumn>
-            <TableHeaderColumn>Department</TableHeaderColumn>
-            <TableHeaderColumn>Last edited</TableHeaderColumn>
-          </TableRow>
-        </TableHeader>
-        <TableBody displayRowCheckbox={false}>
-          <TableRow>
-            <TableRowColumn>
-              <Link to="/asset/6477365c-ff58-11e7-8be5-0ed5f89f718b">
-                HR Records - Staff and Salaries
-              </Link>
-            </TableRowColumn>
-            <TableRowColumn>In-progress</TableRowColumn>
-            <TableRowColumn>University Information Services (UIS)</TableRowColumn>
-            <TableRowColumn>11/01/2018</TableRowColumn>
-          </TableRow>
-          <TableRow>
-            <TableRowColumn>
-              <Link to="/asset/8f35d858-ff58-11e7-8be5-0ed5f89f718b">
-                User behaviour for Research Dashboard
-              </Link>
-            </TableRowColumn>
-            <TableRowColumn>Complete</TableRowColumn>
-            <TableRowColumn>University Information Services (UIS)</TableRowColumn>
-            <TableRowColumn>11/01/2018</TableRowColumn>
-          </TableRow>
-        </TableBody>
-      </Table>
+  	<div className="App-main">
```

Comment:
Can we use an indent of 2 spaces here to be consistent with the rest of the code?